### ISSUE TITLE
Aviewer not working on Windows10

### ISSUE THREAD

**Issue by PavaniHFP** [ThePacielloGroup/WebAccessibilityToolbar#97]
I am performing some tests on Windows 10 machine to see if the WAT toolbar works. Using Windows10/IE11 as soon as i chose the aviewer option I am getting the message as "A problem caused the program to stop working correctly. Windows will close the program and notify you if a solution is available. I am using the WAT version of 2/2015. Can you please confirm if Aviewer has any issues with Windows 10
![a](https://cloud.githubusercontent.com/assets/13559109/8964524/1095ea22-35f4-11e5-9e98-88bd1dfa3588.png)

**Comment by menovak** [comment]
An issue with 'aViewer' on Windows 10 has been reported, but I can't elaborate much at this point.  CCan you confirm/use the other WAT related toolbar functions?

**Comment by PavaniHFP** [comment]
Yep i can use the other tools with in WAT toolbar and it functions fine

**Comment by menovak** [comment]
Thanks for that feedback PavaniHFP

**Comment by matatk** [comment]
I have encountered this problem and found that using the [standalone AViewer](https://github.com/ThePacielloGroup/aviewer/releases/latest) works (thanks @patrickhlauke for the suggestion).  Therefore it seems that we need to either include that version of AViewer here or, more likely given the time constraints, re-release WAT with a link to the latest AViewer, rather than an embedded copy.

Wondering what @menovak, @patrickhlauke and @stevefaulkner think about this course of action?

**Comment by patrickhlauke** [comment]
the question really comes down to: can we get these sorts of final broken things fixed in a timely manner, before WAT is sunsetted officially? need to set aside a chunk of time to brutally triage the remaining open issues at the end of the summer, actually fix what we decide to fix, release the final version and then close development/issues.
